portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,19 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function getComment(){
+    fetch('/data').then(response => response.json()).then((data) => {
+        const CommentListElement = document.getElementById('comment-list');
+        data.forEach((data) => {
+        CommentListElement.appendChild(createCommentElement(data));
```

Comment:
I think this line should be indented which I think the formatter should do for you.